tests/test_magicfields.py

```diff
@@ -4,8 +4,8 @@
 
 from scrapy.spiders import Spider
 from scrapy.utils.test import get_crawler
-from scrapy.item import DictItem, Field
 from scrapy.http import HtmlResponse
+from scrapy.item import Dict as DictItem, Field
```

Comment:
`scrapy.item.Dict` is `typing.Dict`, and it is not meant to be imported like this. The replacement for `DictItem` is `Item`.